tests/aws/templates/cfn_ref_unsupported.yml

```diff
@@ -0,0 +1,10 @@
+Resources:
+  CodeDeployApplication:
+    Type: AWS::CodeDeploy::Application
```

Comment:
nit: I'd suggest using something that we're sure won't be implemented in the future here instead, e.g. `AWS::LocalStack::Unknown`

Reply:
Thanks for the suggestion. I extended the test to use an intrinsic function an a development. Also moved the patch outside the recursive function.